packages/terra-status/src/terra-dev-site/doc/example/StatusArrange.jsx

```diff
@@ -4,14 +4,15 @@ import Arrange from 'terra-arrange';
 /* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
 import Status from 'terra-status/lib/Status';
 import placeholderPic150x150 from 'terra-status/lib/terra-dev-site/doc/example/150x150.jpg';
+import './colors.css';
 /* eslint-enable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
 
 const image = <img style={{ display: 'block' }} height="150" width="150" src={placeholderPic150x150} alt="placeholder" />;
 const simpleText = <div style={{ padding: '5px' }}>Sample text</div>;
 
 const StatusArrange = () => (
   <div>
-    <Status color="#ff0000" visuallyHiddenText="Status High">
+    <Status className="high-status" visuallyHiddenText="Status High">
```

Comment:
```suggestion
    <Status className="attention-status" visuallyHiddenText="Status Attention">
```
(update corresponding class names and text content)

Reply:
Fixed in https://github.com/cerner/terra-core/pull/2211/commits/618ae6178edc4a734ef31a044c3bc8d570051b5c